test/commands/mentor/reply_to_discussion_test.rb

```diff
@@ -31,7 +31,7 @@ class Mentor::ReplyToDiscussionTest < ActiveSupport::TestCase
       "foobar"
     )
     assert_equal 1, user.notifications.size
-    notification = user.notifications.first
+    notification = Notification.where(user: user).first
```

Comment:
These are from me working on getting strict_loading working. It's problematic with FactoryBot so I've stopped for now. But these are still fine to merge.

Reply:
What's strict loading?

Reply:
It basically causes an exception if an `n+1` might happen.

https://github.com/rails/rails/pull/37400
https://github.com/rails/rails/pull/39491